cheroot/test/test_makefile.py

```diff
@@ -51,3 +53,27 @@ def test_bytes_written():
     wfile = makefile.MakeFile(sock, 'w')
     wfile.write(b'bar')
     assert wfile.bytes_written == 3
+
+
+def test_close_is_idempotent():
+    """Test that close() can be called multiple times safely."""
+    raw_buffer = io.BytesIO()
+    buffered_writer = makefile.BufferedWriter(raw_buffer)
+
+    # Should not raise any exceptions
+    buffered_writer.close()
+    buffered_writer.close()  # Second call should be safe
+
+    assert buffered_writer.closed
```

Comment:
This will make it explicit that the second call is performed on an already-closed buffer.
```suggestion
    buffered_writer.close()

    assert buffered_writer.closed

    buffered_writer.close()  # Second call should be safe
```